packages/spanner/test/codec.js

```diff
@@ -453,13 +453,15 @@ describe('codec', function() {
 
     it('should only encode public properties of objects', function() {
       var obj = {
-        hasOwnProperty: function(key) { return key === 'public' },
+        hasOwnProperty: function(key) {  // jshint ignore:line
```

Comment:
I dislike ignoring things in the linter, but this is a spot where it really is the right thing to do.
We _want_ the linter to complain if we define `hasOwnProperty` arbitrarily, but in this spot it is the entire point of the test.